api/src/main/java/jakarta/persistence/GenerationType.java

```diff
@@ -42,7 +42,14 @@
      * Indicates that the persistence provider must assign 
      * primary keys for the entity using a database identity column.
      */
-    IDENTITY, 
+    IDENTITY,
+
+    /**
+     * Indicates that the persistence provider must assign
+     * primary keys for the entity by generating a RFC 4122
+     * Universally Unique IDentifier.
+     */
+    UUID,
```

Comment:
Adding a value to enum in java is always backwards incompatible change. Given the strict requirement on semantic versioning for spec projects, we would need to bump spec version to 4.0 - requires new project plan or wait till there is progress review for current version and bump the spec version during that.